libmscore/rendermidi.cpp

```diff
@@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
             return false;
       }
 
+void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
```

Comment:
the factorization into functions is great. Thank you for making it. As you will see all around the code, every function has an header 
```
//---------------------------------------------------------
//   functionName
//    optional explanation on multiple line if needed
//---------------------------------------------------------
```

Regarding the indentation, we use 6 spaces to indent. (the code is 4 spaces indented in your PR) The { after a functionName are on a new line. We use `if (nospace) {` and idem for while, case etc... I started to reindent it so don't bother for this one.